test/arelastic/aggregations/bucket_sort_test.rb

```diff
@@ -0,0 +1,40 @@
+require 'helper'
+
+class Arelastic::Aggregations::BucketSortTest < Minitest::Test
+  def test_as_elastic
+    with_hash_sort = Arelastic::Aggregations::BucketSort.new('foo_agg', 'size' => 3, 'from' => 5, 'sort' => {
+      'bar_bucket' => 'desc', 'baz_bucket' => 'asc', 'fiz_bucket' => { 'order' => 'asc' }
+    })
+    expected = {
+      'foo_agg' => {
+        'bucket_sort' => {
+          'sort' => [
+            { 'bar_bucket' => 'desc' },
+            { 'baz_bucket' => 'asc' },
+            { 'fiz_bucket' => {'order' => 'asc'} }
+          ],
+          'from' => 5,
+          'size' => 3
+        }
+      }
+    }
+    assert_equal expected, with_hash_sort.as_elastic
+
+    with_array_sort = Arelastic::Aggregations::BucketSort.new('foo_agg', 'sort' =>
+      ['bar_bucket', {'baz_bucket' => 'desc'}, {'fiz_bucket' => {'order' => 'desc'}}]
+    )
+    expected = {
+      'foo_agg' => {
+        'bucket_sort' => {
+          'sort' => [
+            'bar_bucket',
+            { 'baz_bucket' => 'desc' },
+            { 'fiz_bucket' => {'order' => 'desc'} }
+          ]
+
+        }
+      }
+    }
+    assert_equal expected, with_array_sort.as_elastic
+  end
+end
```

Comment:
If you just need to test parse_sort, write a smaller test(s) for that.  I think the current tests are a bit overkill.